### PR TITLE
fix(PS2): allow zero min balance

### PR DESCRIPTION
**Pull request by cwagner22**

### CODE REVIEW

File: packages/ps2/src/util/validation.ts [hunk 1]
```diff
@@ -4,8 +4,8 @@ import BigNumber from 'bignumber.js';
 const inputAmountNumberValidation = yup
   .string()
   .required('common:validation.invalid-amount')
-  .test('int', 'common:validation.invalid-amount', (val) => {
-    return String(val) !== '0.0' && String(val) !== '0' && String(val) !== '0.';
+  .test('int', 'common:validation.invalid-value', (val) => {
+    return String(val)[String(val).length - 1] !== '.';
```
Comment (strrife) [inline packages/ps2/src/util/validation.ts:8]: why? 2. is a completely valid value. Additionally, the before code would not pass for "0.0"

Comment (cwagner22) [reply packages/ps2/src/util/validation.ts:8]: But 0.0 should be allowed and positive number should be checked in the validation below (inputAmountNumberValidation)

Comment (strrife) [reply packages/ps2/src/util/validation.ts:8]: what about 2. ?

Comment (cwagner22) [reply packages/ps2/src/util/validation.ts:8]: it won't be accepted due to the line just below:
```
  .test('max', 'common:validation.invalid-value', function (val) {
    return !new BigNumber(val).isNaN();
  });
```

**Review by strrife — APPROVED** [review]

**Pull request merged**